src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_core.clj

```diff
@@ -40,11 +40,11 @@
         changed-path (when-not (= :unknown kind)
                        (.context event))
         full-path (when-not (= :unknown kind)
-                    (-> watched-path (.resolve changed-path) fs/file))]
+                    (-> watched-path (.resolve changed-path) (.toFile)))]
     {:type kind
      :count (.count event)
-     :watched-path (fs/file watched-path)
-     :changed-path (fs/file changed-path)
+     :watched-path (.toFile watched-path)
+     :changed-path (when changed-path (.toFile changed-path))
```

Comment:
The new format looks good.  My only thought would be to be a bit more explicit with the key names, like `:changed-relative-path` and `:changed-absolute-path`.  

On second thought ... it's actually pretty weird to return two separate `File` objects for a single path on the filesystem.  Which makes me wonder if we could just have a single `:changed-path` which is absolute.  Is there any specific need for the relative path to show up in these Events?

I still wonder if there should be a test to go along with this change, since it was clearly broken before but was not caught by the existing tests.